src/components/Property/Types/NumericProperty/Slider/NumericPropertySlider.tsx

```diff
@@ -0,0 +1,71 @@
+import { MantineStyleProps, NumberFormatter, Slider, Stack } from '@mantine/core';
+
+import { usePropListeningState } from '@/api/hooks';
+import { useSliderScale } from '@/components/Property/SliderUtil/hook';
+
+import { SliderMinMaxLabels } from '../../../SliderUtil/SliderMinMaxLabels';
```

Comment:
We're getting deep on the nesting here, I get that this is technically in the same parent folder. But do we want to use the @ notation here?